app/components/blacklight/gallery/document_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render(Blacklight::DocumentComponent.new(classes: 'col', component: :div, document: @document, counter: @counter)) do |component| %>
   <% component.with(:body) do %>
     <div class="thumbnail-container">
-      <%= thumbnail %>
+      <%= method(:thumbnail).arity.zero? ? thumbnail : thumbnail(class: 'img-thumbnail') %>
```

Comment:
If this seems like a no-go, we could theoretically override `#thumbnail` on the component to inject a class as the optional arg.  Or just say screw it and make whatever release that includes this require BL 7.15 or whatever, but that seems like it would be kinda annoying (but maybe I'm overthinking it)

```ruby
def thumbnail(image_options = { class: 'img-thumbnail' })
  super
end
```